_includes/head.html

```diff
@@ -6,11 +6,11 @@
   <meta name="description" content="">
   <meta name="author" content="">
 
-  <title>Phoxygen.com Corporate website</title>
+  <title>{{page.title}}</title>
 
   <!-- Bootstrap Core CSS -->
   <link href="css/bootstrap.min.css" rel="stylesheet">
-  <link rel="icon" href="img/favicon.png" sizes="16x16 32x32" type="image/png">
+  <link rel="icon" href="img/favicon.ico" sizes="16x16 32x32" type="image/png">
```

Comment:
please update the MIME type then — `image/vnd.microsoft.icon` is the “official”’one, though `image/x-icon` should work as well